src/components/navigation/index.less

```diff
@@ -173,6 +173,11 @@
 
 .nav__social-icons-item-link {
   color: @clr-text;
+  display: flex;
+
+  svg {
```

Comment:
Please follow BEM notation:
```
.nav__social-icons-item-link {
  color: @clr-text;
  display: flex;
  .social-icon {
    margin: auto;
  }
}
```
However, it's not good BEM anyway - we would need to pass the container down to SocialIcons component to make it right

Reply:
yep, you're right, I'm not very familiar with BEM yet, but I'll have a look. Thanks!